test/test_checker.ts

```diff
@@ -6,6 +6,8 @@ import sample from "./fixtures/sample-ti";
 import shapes from "./fixtures/shapes-ti";
 import * as enumUnion from "./fixtures/enum-union";
 import enumUnionTI from "./fixtures/enum-union-ti";
+import * as intersection from "./fixtures/intersection";
```

Comment:
This import is unused as far as I can tell.

Reply:
Removed.